torchvision/prototype/features/_image.py

```diff
@@ -182,8 +195,8 @@ def rotate(
         fill: FillTypeJIT = None,
         center: Optional[List[float]] = None,
     ) -> Image:
-        output = self._F._geometry.rotate_image_tensor(
-            self, angle, interpolation=interpolation, expand=expand, fill=fill, center=center
+        output = self._F.rotate_image_tensor(
```

Comment:
The `_F._geometry` crept in with #6476. I only found this because my regex replace did not handle this case :smile: